content/docs/operator-guide/create-targets.md

```diff
@@ -13,55 +13,66 @@ Before we get started, make sure you've got the following installed:
 
 {{< download-button href="https://store.docker.com/search?type=edition&offering=community" color="blue" >}} Install Docker {{</>}}
 
+## Creating custom images
 
-## Creating a custom stack
-
-In this tutorial we will create a sample stack based on `Ubuntu Bionic`. To create a custom stack, we need to create customized build and run images. Let's see how we can do so!
-
+In this tutorial we will create sample build and run images based on `Ubuntu Jammy.`
 
 ### Create a common base image
 
-Let's start by creating a base image containing layers that will be required by both the `build` and `run` images. In order to do this, switch to a clean workspace and create a `Dockerfile` as specified below:
+Let's start by creating a base image containing layers that will be required both the `build` and `run` images.
+In order to do this, switch to a clean workspace and create a `Dockerfile` as specified below:
 
 #### Defining the base
-We start with `ubuntu:jammy` as our `base` image. Since we will be reusing these layers in both our build and run images we will be defining a common base image and leveraging [Docker's multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to ensure this acts as the common base image for both our build-time and run-time environment.
+We start with `ubuntu:jammy` as our `base` image, and we hope you like jammy too!
+Since we will be reusing these layers in both our build and run images we will be defining a common base image and leveraging [Docker's multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to ensure this acts as the common base image for both our build-time and run-time environment.
 
 ```Dockerfile
 # 1. Set a common base
 FROM ubuntu:jammy as base
 ```
 
+
 #### Set required CNB information
 
-Next, we will be setting up the base image as required by the [Cloud-Native Buildpack specification][stack-spec] noted below.
+Next, we will be setting up the base image as required by the [Cloud-Native Buildpack specification](https://github.com/buildpacks/spec/blob/main/platform.md).
 
 ##### Specification
 
-**Labels**
+The image's config's `os` and `architecture` should be set ot valid values according to the
+[OCI Image Specification](https://github.com/opencontainers/image-spec/blob/main/config.md) with the addition of wildcards: 
+Buildpack images may have their architecture set to `*` to indicate "any" - e.g. a shell script that is expected to succeed in any architecture could specify `*`. 
+Similarly 
+
+**Labels (optional)**
+
+##### TODO / question:   I think the spec says that "the platform"  (as opposed to the build image author) should set the labels?
+https://github.com/buildpacks/spec/pull/335/files#diff-e603760990971da3f77be4bb8d77c3405098f006814fd8c054d2d15f395b8330R199
+should we mention them in this tutorial? 
+##### end TODO /question
+
+
```

Comment:
What direction is this taking?  Can I help here?

Reply:
hmm. this is wrong as -is.   the "image config" doesn't get the `*` but the labels _do_, i think.   maybe I can ask @natalieparellano to confirm

Reply:
ah, so, we said,  heck with it let's just not mention it in the tutorial since it's optional anyways, and only getting more optional as the spec gets closer and closer to being released.